pkg/services/provisioning/plugins/plugin_provisioner_test.go

```diff
@@ -29,24 +29,6 @@ func TestPluginProvisioner(t *testing.T) {
 			return nil
 		})
 
-		bus.AddHandler("test", func(ctx context.Context, query *models.GetPluginSettingByIdQuery) error {
```

Comment:
Feels like we still want to keep some of these asserts to make sure arguments are provided as expected. Any reason removing this?

Reply:
we're not using bus any more, so these are not mocking anything

Reply:
I know, but there are now logic in these handlers together with verifying further down how many calls (commands) was dispatched and what the properties of that call was. Now when we have a service instead can maybe do similar things was my idea.

Reply:
Maybe not that important? Just want to make sure we have test coverage of the most important things

Reply:
this is testing the dispatch a method from a different file, that;s why I don;t think it should be tested here